packages/cli/internal/pkg/cli/logs_engine.go

```diff
@@ -35,6 +50,27 @@ func (o *logsEngineOpts) Validate() error {
 		return err
 	}
 
+	ctxMap, err := o.ctxManager.List()
+	if err != nil {
+		return err
+	}
+
+	summary := ctxMap[o.contextName]
+	engine := summary.Engines[0].Engine
+
+	if o.workflowRunId == "" {
+		if !summary.IsServerProcessEngine() {
+			log.Warn().Msgf("DEPRECATION WARNING!!")
```

Comment:
Are we going to deprecate this? Full engine log has parts related to engine work itself (at least for Cromwell).

Reply:
For the "serverless" engines yes that's why the warning is inside the if statement.